device/mellanox/x86_64-mlnx_msn4700-r0/plugins/psuutil.py

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/plugins/psuutil.py
```

Comment:
need to verify if the layout is exactly as 2700. if not, create a new file dedicated for this system.

Reply:
verified during the bring up test.